Pipfile

```diff
@@ -0,0 +1,18 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+python-dotenv = "*"
+pylint = "*"
+
+[packages]
+flask = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
Well... então, fui pego de surpresa por isso aqui kkkkkkkkk esse danado desse requires dá um problema no meu pc, que nn tem python 3.6, mas sim o 3.7, porque ele não acha o 3.6 e por isso não dá o pipenv install, eu tenho de definir onde tá o python que usarei,  daí eu vi [aqui](https://docs.pipenv.org/en/latest/basics/#specifying-versions-of-python) que tem uma observação dizendo `feel free to simply remove this section` KKKKKKKKK só que remover isso vai fazer com que no nosso `pipenv install` inicial (o que cria o virtualenv e baixa as coisas inciialmente), seja agora `pipenv install --three`, pq ele não vai ter mais no pipfile indicando qual versão do python colocar no virtualenv, daí a gente tem de definir na hora que ele cria, qual é o python que ele vai usar... Assim, não é um grande problema, só lembrar na hora de botar no readme, que pra configurar, tem de rodar inicialmente `pipenv install --three` ao invés de só `pipenv install`... anyway, só vai ser obrigatório o uso do `--three` em computadores que não tem python3 como default, (ou seja, PCs do lcc2... por isso é bom previnir e deixar isso como step default de configuração no readme e garantir que vai funcionar em todo canto)

Reply:
Uma coisa que eu vi agora, é que ao invés de ="3.6" dá pra botar só "3", só tem de ver se fica tudo ok assim nas máquinas do lcc2, aqui no meu resolveu o problema

Reply:
Acabei de testar no meu PC e se colocar apenas o 3 funcionou de boas, e a versão do Python que roda no meu PC é a mesma dos PCs do LCC2, ou seja, a `3.6`. Creio que fazer essa alteração não vai impactar negativamente.